examples/demo/src/App.js

```diff
@@ -8,10 +8,22 @@ function App() {
   return (
     <div>
       <h1>DocSearch v3 - React</h1>
+      <button data-testid="btn">A button</button>
       <DocSearch
         indexName="docsearch"
         appId="R2IYF7ETH7"
         apiKey="599cec31baffa4868cae4e79f180729b"
+        transformItems={
+          (items) => { // transform absolute url into relative ones to solve CORS problem in cypress
```

Comment:
The api would return urls of different origin from `localhost` which would result in a CORS error in cypress, so I transform the results. It's a simplify version of how docusaurus makes it https://github.com/facebook/docusaurus/blob/0985fa0af338068b4906f409a3f874a319f5359c/packages/docusaurus-theme-search-algolia/src/theme/SearchBar/index.tsx#L175

Reply:
Can we maybe wrap it in an env check? So it's only replaced for the tests

Or you can copy the whole demo folder and name it like `cypress`

Reply:
Good idea, I'd go with another folder so we can avoid messing up with other two demo.